api-list.go

```diff
@@ -319,7 +360,7 @@ func (c Client) ListObjects(bucketName, objectPrefix string, recursive bool, don
 		var marker string
 		for {
 			// Get list of objects a maximum of 1000 per request.
```

Comment:
What is the maximum returned list size per request? Is it 10000?

Reply:
maximum depends on server implementation @kannappanr